portal-impl/src/com/liferay/portal/model/impl/UserImpl.java

```diff
@@ -737,7 +737,9 @@ public boolean hasMySites() throws PortalException {
 		if ((PrefsPropsUtil.getBoolean(
 				CompanyThreadLocal.getCompanyId(),
 				PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED) ||
-			 PropsValues.LAYOUT_USER_PUBLIC_LAYOUTS_ENABLED) &&
+			 PrefsPropsUtil.getBoolean(
+				 CompanyThreadLocal.getCompanyId(),
```

Comment:
@torresvictor100, can you please replace by the method `getCompanyId()`? I realized that this method is available on an abstract class implemented by `UserImpl`. Please use this [code](https://github.com/natocesarrego/liferay-portal/blob/b7589e8a98e5dad79ebcac01493994ff474cfd85/portal-impl/src/com/liferay/portal/model/impl/UserImpl.java#L167) as an example.

You did a good job here and also followed our agreement, but, for this specific class, I think we can reuse what is already available on it, this way we reduce the probability to introduce issues to the code. :wink: